base/database/testing.go

```diff
@@ -442,3 +442,8 @@ func CheckBaselineDeleted(t *testing.T, baselineID int) {
 	assert.Nil(t, Db.Model(&models.SystemPlatform{}).Where("baseline_id = ?", baselineID).Count(&cntSystems).Error)
 	assert.Equal(t, 0, int(cntSystems))
 }
+
+func GetAllSystems(t *testing.T) (systems []*models.SystemPlatform) {
+	assert.Nil(t, Db.Model(&models.SystemPlatform{}).Select("*").Order("rh_account_id").Scan(&systems).Error)
+	return systems
```

Comment:
Select() is redundant, `Db.Model(&models.SystemPlatform{}).Order("rh_account_id").Scan(&systems).Error` is enough.